packages/backend/src/api/controllers/events/EventsController.ts

```diff
@@ -71,30 +61,16 @@ export class EventsController {
   }
 }
 
-function getEventChart(
-  records: EventRecord[],
-  eventNames: string[],
-): EventChart {
-  //Record<timestamp, Record<eventName, count>>
-  const entries: Record<number, Record<string, number> | undefined> = {}
-
-  for (const { timestamp, name, count } of records) {
-    const entry = entries[timestamp.toNumber()] ?? {}
-    entry[name] = count
-    entries[timestamp.toNumber()] = entry
-  }
-
-  const data: EventChartPoint[] = []
-
-  for (const key in entries) {
-    data.push([
-      new UnixTime(Number(key)),
-      ...eventNames.map((e) => entries[key]?.[e] ?? 0),
-    ])
-  }
-
-  return {
-    types: ['timestamp', ...eventNames],
-    data,
-  }
+export function getEndpointConfig(timestamp: UnixTime | undefined): {
+  granularity: EventGranularity
+  timestamp: UnixTime | undefined
+}[] {
+  return [
+    { granularity: 'hourly', timestamp: getHourlyMinTimestamp(timestamp) },
```

Comment:
Maybe `getHourlyMinTimestamp` and `getSixHourlyMinTimestamp` should be moved to `/controllers/utils` since they are used in two different contexts